### DISCUSSION TITLE
Does Zitadel provides SSO page like Authentik does?

### DISCUSSION THREAD

**Discussion by neox8** [zitadel/zitadel#4901] [Ideas]
![image](https://user-images.githubusercontent.com/6768920/208409156-f5b71a0a-74d4-4b62-ba70-0ca25685b007.png)

something like above - thanks!

**Comment by fforootd** [comment]
Currently not, even though the information is available.

I think there where others also interested in such a kind of "portal" functions, I just do not find the issue anymore 😢 

At the moment nothing is planed to expand towards this, although we would be open for contributions.
